optimism/src/keccak/tests.rs

```diff
@@ -199,15 +199,37 @@ fn test_keccak_fake_witness_wont_satisfy_constraints() {
     // NEGATIVIZE THE WITNESS
 
     // Break padding constraints
-    witness_env[0].witness[KeccakColumn::PadBytesFlags(0)] = Fp::from(1u32);
-    witness_env[0].constrain_padding();
+    assert_eq!(witness_env[0].is_pad(), Fp::one());
+    // Padding can only occur in suffix[3] and suffix[4] because length is 100 bytes
+    assert_eq!(witness_env[0].pad_suffix(0), Fp::zero());
```

Comment:
And these assertions are not directly related to the issue, just some extra checks?